packages/pluggableWidgets/html-node-web/src/HTMLNode.tsx

```diff
@@ -0,0 +1,67 @@
+import { ReactElement, createElement, ReactNode, Fragment, DOMAttributes, HTMLAttributes } from "react";
+
+import { HTMLNodeContainerProps } from "../typings/HTMLNodeProps";
+import { prepareAttributes, prepareEvents } from "./utils/props-utils";
+
+export function HTMLNode(props: HTMLNodeContainerProps): ReactElement | null {
+    const tag = props.tagName !== "__customTag__" ? props.tagName : props.tagNameCustom;
+    if (props.tagUseRepeat) {
+        const items = props.tagContentRepeatDataSource?.items;
+        if (!items?.length) {
+            return null;
+        }
+
+        return (
+            <Fragment>
+                {items.map(item => (
+                    <HTMLTag
```

Comment:
what I am thinking about this part is that it confuses me a little bit, we have duplication, which we can handle here.|
I do see a few options on how to do that.

let's move `HTMLTag` component call into a function `const renderNode = (someProps) => <HTMLTag/>`, and pass props there.
Or maybe push everything in one format into a separate array "even if it's not repeatable action" and render the final HTMLTag component.
Like always return ```{items.map(item => (<HTMLTag {...item}/>)```

Reply:
Yes, good suggestion! I pushed quite some changes, please recheck